korgan/robust_production_legal.py

```
@@ -179,8 +180,11 @@ async def _structured_response(
             "prompt_cache_key": f"korgan:{schema_name}:v4",
         }
 
-        if model == "gpt-5.1" or model.startswith("gpt-5.1-"):
-            kwargs["reasoning"] = {"effort": "none"}
+        # Служебные вызовы остаются без рассуждения, составление документа его
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `е` (CYRILLIC SMALL LETTER IE). Did you mean `e` (LATIN SMALL LETTER E)?

(RUF003)